build/test-requirements.txt

```diff
@@ -1,5 +1,6 @@
 flake8
-jaxlib==0.1.51
+ # Must be kept in sync with the minimum jaxlib version in jax/lib/__init__.py
```

Comment:
Isn't the minimum version 1.51 still? What I would do is to bump the minimum version (and the version here) in a few days, unless we discover a problem with 1.55

Reply:
I see you've done it below. Ok.